services/docker-compose-dev-vendors.yml

```diff
@@ -14,10 +14,10 @@ services:
         - io.simcore.zone=${TRAEFIK_SIMCORE_ZONE}
         - traefik.enable=true
         - traefik.docker.network=${SWARM_STACK_NAME}_default
-        # auth
+        # auth: https://doc.traefik.io/traefik/middlewares/http/forwardauth
         - traefik.http.middlewares.${SWARM_STACK_NAME}_manual-auth.forwardauth.address=http://${WEBSERVER_HOST}:${WEBSERVER_PORT}/v0/auth:check
         - traefik.http.middlewares.${SWARM_STACK_NAME}_manual-auth.forwardauth.trustForwardHeader=true
-        - traefik.http.middlewares.${SWARM_STACK_NAME}_manual-auth.forwardauth.authResponseHeaders=Set-Cookie,osparc-sc
+        - traefik.http.middlewares.${SWARM_STACK_NAME}_manual-auth.forwardauth.authResponseHeaders=Set-Cookie,osparc-sc,X-Simcore-Products-Name
```

Comment:
Please try and run any dynamic-service and you will see that it will no longer work.
Also you are breaking the "notebook" feature of s4l currently, since it opens in a different tab.

Reply:
i forgot thisone. thx